schemas/includes/unknown_entity_type.json

```diff
@@ -52,6 +52,12 @@
     "mailing_address": {
       "$ref": "address.json"
     },
+    "identifiers": {
+      "type": ["array", "null"],
```

Comment:
I was following other examples of identifiers (e.g. in company schema). Can easily change, but we potentially have a choice between inconsistency vs breaking existing bots